dockerclient_test.go

```diff
@@ -143,6 +145,49 @@ func TestContainerLogs(t *testing.T) {
 	}
 }
 
+func TestContainerStats(t *testing.T) {
+	client := testDockerClient(t)
+	var expectedContainerStats stats.Stats
+	if err := json.Unmarshal([]byte(statsResp), &expectedContainerStats); err != nil {
+		t.Fatalf("cannot parse expected resp: %s", err.Error())
+	}
+	containerIds := []string{"foobar", "foo"}
+	expectedResults := [][]interface{}{
+		{expectedContainerStats, "error"},
+		{expectedContainerStats, expectedContainerStats},
+	}
+	errored := false
+
+	for i := range containerIds {
+		fmt.Printf("on big iter %d\n", i)
```

Comment:
If you plan on keeping these prints, consider using `t.Log()`/`t.Logf()` instead.